python/pyspark/rdd.py

```diff
@@ -70,7 +70,8 @@ def portable_hash(x):
     >>> portable_hash((None, 1)) & 0xffffffff
     219750521
     """
-    if sys.version >= '3.3' and 'PYTHONHASHSEED' not in os.environ:
+
+    if sys.version >= '3.2.3' and 'PYTHONHASHSEED' not in os.environ:
```

Comment:
I would avoid the string comparison here. If you must, you can use `distutils.version.StrictVersion`

```
In [15]: StrictVersion('3.10.0') > StrictVersion('3.2.3')
Out[15]: True
```

but `sys.version` will break anyway on some Python distributions.

```
In [16]: print(sys.version)
3.4.4 |Continuum Analytics, Inc.| (default, Jan  9 2016, 17:30:09)
[GCC 4.2.1 (Apple Inc. build 5577)]
```

You might want to use `sys.version_info` instead.